.browserslistrc

```diff
@@ -0,0 +1,3 @@
+last 4 versions
```

Comment:
Both `browserslist` and `.browserslistrc` are supported, but the first one crashes Cypress 🤷